content/html-css/sprints/1/prep/index.md

```diff
@@ -7,11 +7,40 @@ weight = 1
 backlog= 'Module-HTML-CSS'
 backlog_filter= 'Week 1'
 [[blocks]]
+name="spaced-repetition"
+[[blocks]]
+name="what-is-html"
+src="module"
+[[blocks]]
 name="Transferable skills"
 src="https://cyf-pd.netlify.app/blocks/prep-transferable-skills/readme/"
 [[blocks]]
 name="Feedback"
 src="https://cyf-pd.netlify.app/blocks/prep-for-feedback/readme/"
+[[blocks]]
+name="Playlist"
+src="https://www.youtube.com/watch?v=mBmX7944vd0&list=PLozA7cloMbPgk5gK8His2CeuJTKofypbV&index=7"
+[[blocks]]
+name="Bikes for Refugees"
+src="https://github.com/CodeYourFuture/bikes-for-refugees/readme"
+[[blocks]]
+name="Prepare for the Devtools workshop"
```

Comment:
I'm finding the learning objectives displayed at the start of each section in the workshop pretty confusing (and I know what they are there for...) - can we either give them a heading like "Learning objectives" or hide them behind a details/summary box by default? I keep thinking they're things I need to do as prep, realising I don't understand all of the words in them, and then realising they're meant to be the outcomes I'm taking away from the section...

Reply:
They can go in a details box. Make a ticket?